tests/test_jobwatch.py

```diff
@@ -4,17 +4,58 @@
 
 sys.path.insert(0, os.path.join(os.path.dirname(__file__), '..'))
 import jobwatch
+# import skawatch
+
+# Ska-specific watchers
```

Comment:
Why are the classes copied into the testing?

Reply:
Because I wanted to leave `skawatch.py` as a script which is not importable.  I actually did change it to an importable module with a `main()`, but then all the diffs were invisible.

Reply:
OK. If you're happy with this, fine... it just means if we change something in one of the classes as they are actually used, that won't be the thing that is tested, right? I don't know if that is worse.

Reply:
This will be easier to do In the next round when this whole things gets converted to a package. It's fine now,